container/layer_tools.bzl

```diff
@@ -120,8 +120,10 @@ def get_from_target(ctx, name, attr_target, file_target = None):
     else:
         if not hasattr(attr_target, "files"):
             return {}
-        target = attr_target.files.to_list()[0]
-        return _extract_layers(ctx, name, target)
+        tar_files = [file for file in attr_target.files.to_list() if file.extension == "tar"]
```

Comment:
so the bug was the toolchain was giving us more than one file?

Reply:
The `rule_test` was providing the files listed in `generates`. The issue was that this code assumed that there was either one file or that the first file is a tar file. So the fix I went with was to search for tar files and require that only one was found.

Reply:
The issue originates from 
https://github.com/bazelbuild/rules_docker/blob/0adf8b2ff23e8d7a14562be0f5707cd4dbb32998/tests/docker/util/BUILD#L82-L85